frontend/src/component/UserForm.tsx

```diff
@@ -0,0 +1,51 @@
+import {Key, Person} from "react-bootstrap-icons";
```

Comment:
Cool. Also take a look at these great [icons](https://fontawesome.com/).

Reply:
Thanks for the tip! I will check them out :)